docs/src/main/sphinx/connector/redshift.md

```diff
@@ -255,3 +218,41 @@ FROM
 
 ```{include} query-table-function-ordering.fragment
 ```
+
+## Performance
+
+The connector includes a number of performance improvements, detailed in the
+following sections.
+
+### Parallel read via S3
+
+The connector supports the Redshift `UNLOAD` command to transfer data to Parquet
+files on S3. This enables parallel read of the data in Trino instead of the
+default, single-threaded JDBC-based connection to Redshift, used by the
+connector.
+
+Configure the required S3 location with `redshift.unload-location` to enable the
+parallel read, and define a custom life cycle policy on the S3 bucket. Parquet
+files are automatically removed with query completion. The Redshift cluster and
+the configured S3 bucket must use the same AWS region.
+
+:::{list-table} Parallel read configuration properties
```

Comment:
@mayankvadariya Is `unload_enabled` session property intentionally undocumented?

Reply:
It should be documented

Reply:
I added it .. but really it can only be used to deactivate it right?